doc/explanation/configure-storage.rst

```diff
@@ -173,4 +173,4 @@ with desired parameters, and then select these partitions or devices as mount
 points in the installer. Any changes you make while the installer is running
 but before altering the storage configuration will reflected in the installer.
 
-The installer cannot yet configure iSCSI mounts or btrfs subvolumes.
+The installer cannot yet configure iSCSI mounts or Btrfs subvolumes.
```

Comment:
Looking at https://btrfs.readthedocs.io/en/latest/search.html?q=btrfs&check_keywords=yes&area=default it appears that the BTRFS project prefers the all upper case form.  That search result shows things as inconsistent though.  WDYT?

Reply:
Yers, BTRFS is better.